backend/src/scripts/migrations/5.2/moveNotificationsToInternal.js

```diff
@@ -26,19 +26,27 @@ const NOTIFICATIONS_DB = 'notifications';
 const NOTIFICATIONS_COLL = 'notifications';
 
 const processCollection = async (user) => {
-	const migrationOps = (await find(NOTIFICATIONS_DB, user, {})).map((entry) => ({
-		updateOne: {
-			filter: { _id: entry._id },
-			update: { $set: {
-				...entry,
-				user,
-				timestamp: new Date(entry.timestamp),
-			} },
-			upsert: true,
-		},
+	const updatedNotifications = (await find(NOTIFICATIONS_DB, user, {})).map((entry) => ({
```

Comment:
Probably better to do this
```suggestion
	const updatedNotifications = (await find(NOTIFICATIONS_DB, user, {})).map(({timestamp, ...entry}) => ({
```